packages/cms-lib/lib/config.js

```diff
@@ -242,9 +261,15 @@ const configFileIsBlank = () => {
   return _configPath && fs.readFileSync(_configPath).length === 0;
 };
 
-const createEmptyConfigFile = () => {
-  if (!_configPath) {
-    setDefaultConfigPath();
+const createEmptyConfigFile = ({ deleteOnExitIfBlank }) => {
+  setDefaultConfigPathIfUnset();
+
+  if (configFileExists()) {
+    return;
+  }
+
+  if (deleteOnExitIfBlank) {
+    process.on('exit', deleteEmptyConfigFile);
```

Comment:
This seems like side effect code for a lib function.  Couldn't the subscription be done from the setup code in the init.js command file?

Reply:
I had thought allowing this option would make the method more robust, but I can see how it adds unnecessary complexity to it. Moved the logic out back into init.

Reply:
Thanks for doing that. I think in general we want to minimize ties in `cms-lib` to execution context meaning that we shouldn't assume that it is used in a script. I suspect tackling #4 and the refactoring required could further decouple configuration handling from the CLI/script use case.